tests/axios.test.ts

```diff
@@ -20,7 +22,7 @@ describe('axios instance', () => {
 
   it('initializes with the correct config values', () => {
     jest.isolateModules(() => {
-      const axios = require('axios');
+      const axios: AxiosStatic = require('axios');
```

Comment:
What's the difference between the types here?

Reply:
Oh this I can remove since I didn't end up needing my tests in here. The type without it would be `any` so when I had tests in here I put that on there so `axiosCreateSpy` would be typed properly. It's a good change IMO, but technically isn't needed for this PR. Up to you.